front/src/services/aleph.ts

```diff
@@ -0,0 +1,121 @@
+import { AuthenticatedAlephHttpClient } from "@aleph-sdk/client";
+import { ETHAccount, getAccountFromProvider, importAccountFromPrivateKey } from "@aleph-sdk/ethereum";
+import { ForgetMessage, ItemType, StoreMessage } from "@aleph-sdk/message";
+import { PrivateKey } from "eciesjs";
+import { SignMessageReturnType } from "viem";
+import web3 from "web3";
+import { z } from "zod";
+
+import env from "@/config/env";
+
+// Aleph keys and channels settings
+const SECURITY_AGGREGATE_KEY = "security";
+const BEDROCK_GENERAL_CHANNEL = "bedrock";
```

Comment:
We could put this in env variables instead of hardcoding 
```suggestion
const SECURITY_AGGREGATE_KEY = env.ALEPH_SECURITY_AGGREGATE_KEY;
const BEDROCK_GENERAL_CHANNEL = env.ALEPH_BEDROCK_GENERAL_CHANNEL;
```

Reply:
for the channel why not, but security is a reserved keyword so we don't want to ever change it

Reply:
cf https://docs.aleph.im/protocol/permissions/#the-security-aggregate